github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-periodics.yaml

```diff
@@ -0,0 +1,39 @@
+periodics:
+- name: periodic-kvp-push-nightly-build-main
+  cron: "2 2 * * *"
+  decorate: true
+  annotations:
+    testgrid-dashboards: kubevirt-velero-plugin-periodics
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-gcs-credentials: "true"
+    preset-shared-images: "true"
+    preset-kubevirtci-quay-credential: "true"
```

Comment:
ditto

Reply:
the jon only uses kubevirtci quay as a read only, so I suppose not needed